modules/round/src/main/RoundDuct.scala

```diff
@@ -74,10 +74,12 @@ private[round] final class Round(
 
     case ResignForce(playerId) => handle(playerId) { pov =>
       (pov.game.resignable && !pov.game.hasAi && pov.game.hasClock) ?? {
-        socketMap.ask[Boolean](pov.gameId)(IsGone(!pov.color, _)) flatMap {
-          case true if !pov.game.variant.insufficientWinningMaterial(pov.game.board, pov.color) => finisher.rageQuit(pov.game, Some(pov.color))
-          case true => finisher.rageQuit(pov.game, None)
-          case _ => fuccess(List(Event.Reload))
+        pov.forceResignable ?? {
```

Comment:
Should `forceResignable` be brought up to the prior line?

```
case ResignForce(playerId) => handle(playerId) { pov =>
      (pov.game.resignable && !pov.game.hasAi && pov.game.hasClock && pov.forceResignable)** ?? {
```

Reply:
Semantically there is no difference, personally I prefer the extra line to make it a bit more clear "can we resign this game at all?" => "can we FORCE resign this game?" => ...